pyproject.toml

```diff
@@ -8,6 +8,7 @@ packages = [{include = "sae_lens"}]
 
 [tool.poetry.dependencies]
 python = "^3.10"
+neuron-explainer = { git = "https://github.com/hijohnnylin/automated-interpretability.git", subdirectory = "neuron-explainer" }
```

Comment:
I don't think we can have dependencies on a git repo while still being publishable on PyPI. This would also need to be published on PyPI as well.

Reply:
ah thanks! converting this PR to draft until i do this

Reply:
alrighty it's done and ready for review